test/testsets.json

```diff
@@ -14,6 +14,7 @@
     { "name": "test_dbus_after_destroy.js" },
     { "name": "test_dbus_define.js" },
     { "name": "test_dbus_introspectable.js" },
+    { "name": "test_dbus_send_utf8.js.js" },
```

Comment:
```suggestion
    { "name": "test_dbus_send_utf8.js" },
```

Reply:
Fixed, now it gets green result in my local machine :)